services/QuillLMS/app/models/question.rb

```diff
@@ -48,6 +48,7 @@ class Question < ActiveRecord::Base
   validates :question_type, presence: true, inclusion: {in: TYPES}
   validates :uid, presence: true, uniqueness: true
   validate :data_must_be_hash
+  validate :validate_sequences
 
```

Comment:
In going through this commit I noticed a related method in question.rb  `def stored_as_array`.  I think it would be worth renaming to `stored_as_array?`.